lib/printers/pretty.js

```diff
@@ -55,6 +55,24 @@ class PrettyPrinter {
         this.console.log(chalk.red(chalk.bold(fixableSummary)));
       }
     }
+
+    if (this.options.updateTodo && todoInfo) {
```

Comment:
Unless I'm mistaken, this probably should be `todoInfo.added > 0`. Otherwise, we would print the message `0 todos created`.

Is that what we want? (I'm not sure)

Reply:
Well if you explicitly use `--update-todo` and get no output, even if no todos are created, is that what we want/correct? I would say that outputting explicitly `0 Todos created` seems appropriate to me in this case. WDYT? 

If you agree, I should add a test case for this.

Reply:
Ya, that seems good to me (to basically **always** print something about todos whenever `--update-todo` is used).

Reply:
👍